### DISCUSSION TITLE
How to: two panes only via command line

### DISCUSSION THREAD

**Discussion by user908348** [WinMerge/winmerge#2220] [Q&A] [answered]
When I do a `WinMergeU.exe file1.txt file2.txt` I get three panes, but I actually only want two panes, left & right - not left/middle/right - how to do this via the command line (or Gui for that matter as all I seem to be able to manage to always get three)

**Comment by sdottaka** [comment, marked as answer]
You can display two panes using the command line as you described.

Example:

~~~
"c:\Program Files\winmerge\WinMergeU.exe" C:\Windows\System32\WindowsPowerShell\v1.0\de-DE\default.help.txt C:\Windows\System32\WindowsPowerShell\v1.0\en-US\default.help.txt
~~~

Please provide the actual command line you have written.

**Reply by user908348** [reply]
Thank you, I uninstalled / reinstalled WinMerge and created a new batch file and it now indeed only shows two panels, I'm sure something was wrong earlier but it seems to have resolved. This can be closed (or removed).
